Sources/Sentry/SentryProfiler.mm

```diff
@@ -60,6 +60,19 @@
     return [symbolNSStr substringWithRange:[match rangeAtIndex:1]];
 }
 
+NSString *
+profilerStopReasonName(SentryProfilerStopReason reason)
+{
+    switch (reason) {
+    case SentryProfilerStopReasonNormal:
+        return @"normal";
+    case SentryProfilerStopReasonAppMovedToBackground:
+        return @"backgrounded";
```

Comment:
`l`: ?
```suggestion
        return @"background";
```

Reply:
We already agreed on this spec with android and backend so let's just leave it.